index.js

```diff
@@ -4,7 +4,12 @@ const isPng = require('is-png');
 const optipng = require('optipng-bin');
 
 module.exports = opts => buf => {
-	opts = Object.assign({optimizationLevel: 3}, opts);
+	opts = Object.assign({
+		optimizationLevel: 3,
+		enableBitDepthReduction: true,
```

Comment:
I asked you to drop the `disable` prefix, not change it to `enable`. Should be `bitDepthReduction`. Same with the others.